lib/redmine/menu_manager/top_menu_helper.rb

```diff
@@ -192,16 +205,17 @@ def render_module_top_menu_node(item_groups = module_top_menu_item_groups)
   end
 
   def render_menu_item_group(menu, item_group)
```

Comment:
Please rename this method, since we are not rendering a menu any more. 
```suggestion
  def render_dialog_item_group(dialog, item_group)
```